scripts/openbsd/init.sh

```diff
@@ -42,7 +45,7 @@ fi
 # XXX this patch should be applied in our ports tree, but we desparately need
 # package builder first. remove this when the builder is ready.
 #
-if [ `ansible --version | head -n 1 | cut -d' ' -f 2` == '2.3.2.0' ]; then
+if [ `ansible --version | head -n 1 | cut -d' ' -f 2` == '2.3.2.0' -a `uname -r` != '6.2' ]; then
```

Comment:
ansible package is installed from our ports tree on `6.2` and the patch is necessary.
or exclude `6.2` before the line 15 `sudo pkg_delete ansible`?